Eplant/UI/LeftNav/index.tsx

```diff
@@ -36,40 +59,109 @@ export function LeftNav(props: {
   }, [geneticElements])
   return (
     <Stack gap={2} direction='column' height={'100%'}>
-      <LogoWithText text='ePlant' />
-      <SearchGroup
-        addGeneticElements={(s) => {
-          setGeneticElements(
-            geneticElements.concat(
-              _.uniqBy(
-                s.filter(
-                  (g) => !geneticElements.find((gene) => g.id == gene.id)
-                ),
-                (a) => a.id
-              )
-            )
-          )
-          if (s.length > 0) {
-            setActiveGeneId(s[0].id)
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+        {isCollapse ? <LogoWithText text='' /> : <LogoWithText text='ePlant' />}
+        <div
+          style={
+            isCollapse
+              ? {
+                  cursor: 'pointer',
+                  height: '25px',
+                  width: `${tabWidth}`,
+                  marginRight: '-39px',
+                  backgroundColor: `${theme.palette.primary.main}`,
+                  borderRadius: '20px 0px 0px 20px',
+                  transform: 'translateX(-30%)',
+                  transition: 'all 0.5s ease-out',
+                }
+              : {
+                  cursor: 'pointer',
+                  height: '25px',
+                  width: `${tabWidth}`,
+                  marginRight: '-20px',
+                  backgroundColor: `${theme.palette.primary.main}`,
+                  borderRadius: '20px 0px 0px 20px',
+                  transform: 'translateX(20%)',
+                  transition: 'all 0.5s ease-out',
+                }
           }
-        }}
-      />
-      <Collections
-        onSelectGene={props.onSelectGene}
-        selectedGene={props.selectedGene}
-      />
-      <Box flexGrow={1} />
-      <FormGroup>
-        <FormControlLabel
-          control={
-            <Switch
-              onChange={(v) => setDarkMode(v.target.checked)}
-              checked={darkMode}
+          onClick={settingCollapse}
+          onMouseEnter={() => setTabWidth('40px')}
```

Comment:
This looks great on the UI!

Reply:
Thank you! I removed the package-lock.json and regenerated it via npm install